src/Datasource/QueryTrait.php

```diff
@@ -276,6 +276,10 @@ public function aliasFields(array $fields, ?string $defaultAlias = null): array
     public function all(): ResultSetInterface
     {
         if ($this->_results !== null) {
+            if (!$this->_results instanceof ResultSetInterface) {
```

Comment:
Should this instance of call have parens on it?

Reply:
The parens aren’t needed. The order of operations is the instance of first. 

We could add them. Not sure if phpcs complains

Reply:
The parenthesis is not required but I have added it for better readability.